heim-process/src/sys/windows/bindings/token.rs

```diff
@@ -0,0 +1,66 @@
+use std::ptr;
+use winapi::shared::minwindef::{DWORD, LPVOID};
+use winapi::um::handleapi::CloseHandle;
+use winapi::um::processthreadsapi::OpenProcessToken;
+use winapi::um::securitybaseapi::GetTokenInformation;
+use winapi::um::winnt::{TokenUser, HANDLE, TOKEN_QUERY, TOKEN_USER};
+
+use heim_common::prelude::*;
+use heim_common::sys::windows::Handle;
+use heim_common::Result;
+use heim_host::os::windows::UserExt;
+use heim_host::User;
+
+pub struct Token(HANDLE);
+
+impl Token {
+    pub fn open(process_handle: &Handle) -> Result<Self> {
+        let mut token_handle: HANDLE = ptr::null_mut();
```

Comment:
This `token_handle` should be wrapped into the safe `heim_common::sys::windows::Handle` wrapper as soon as possible and `Token` should carry it instead.